Plugin/Job/Product.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace JustBetter\AkeneoBundle\Plugin\Job;
+
+use Akeneo\Connector\Helper\Config as ConfigHelper;
+use Akeneo\Connector\Job\Product as AkeneoProduct;
+use Akeneo\Connector\Model\Source\Filters\Family;
+use Akeneo\Connector\Model\Source\Filters\Mode;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+
+class Product
+{
+    public const PRODUCTS_FILTERS_EXCLUDED_FAMILIES = 'akeneo_connector/products_filters/excluded_families';
+
+    public function __construct(
+        protected ConfigHelper $configHelper,
+        protected ScopeConfigInterface $scopeConfig,
+        protected Family $familyFilter
+    ) {
+    }
+
+    public function getFamiliesToExport(): ?string
+    {
+        return $this->scopeConfig->getValue(self::PRODUCTS_FILTERS_EXCLUDED_FAMILIES);
+    }
+
+    public function aroundGetFamiliesToImport(
```

Comment:
You can convert this to an after Plugin. Check if array is empty and if there are families to exclude then fetch all families to filter.